web/modules/custom/collation_fixer/README

```diff
@@ -0,0 +1,41 @@
+# Collation Fixer
+
+A 'runtime' requirements checker (admin/reports/status) for checking
+all database tables have the right database collation (probably very
+MySQL centric).
+
+Offers the ability to fix tables with wrong database collation.
+
+The collation is either specified in your database settings.
+
+This can be useful when developing non-english sites.
+
+Example of specifying in database settings in settings.php (will
+affect all tables - it's probably better to do it in hook_schema() or
+hook_schema_alter()):
+
+  $databases = array(
+    'default' => array(
+      'default' => array(
+        'database' => 'my-db',
+        'username' => 'my-user',
+        'password' => 'my-password',
+        'host' => 'localhost',
+        'port' => '',
+        'driver' => 'mysql',
+        'prefix' => '',
+        'collation' => 'utf8_danish_ci',
```

Comment:
Just for clarity I think it would be an idea to use the generally recommended collation here: utf8mb4_danish_ci (which you also make use of in [code](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1156/files#diff-250320d2d3a0d27d487a59fb70ff9b2ef2a008b7e18a49caf7a9d21fc854e9e2R60))